README.md

```
@@ -34,38 +34,39 @@ You need to clone the repository for the new Task. After cloning, move to that f
 
-`dotnet new frendstasktemplate --FullTaskName Frends.ClassName.TaskName --allow-scripts yes`
+`dotnet new frendstasktemplate -F Frends.ClassName.MethodName -D "Description of the Task"`
 
 ## Get help using the template
 
 `dotnet new frendstasktemplate -h`
 
-Example output:
+### Example output:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)